stor/cli.py

```diff
@@ -72,10 +72,14 @@
 Direct file transfer between OBS services or within one OBS service (server-side copy)
 is not yet supported.
 """
+from future import standard_library
+standard_library.install_aliases()
```

Comment:
Note that this introduces global monkey-patching, which is a fairly invasive thing for a dependency to do.